apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -1,24 +1,88 @@
+import { config } from "~/libs/modules/config/config.js";
+import { type EmailService } from "~/libs/modules/email-service/email-service.module.js";
+import { getHtmlMessage } from "~/libs/modules/email-service/libs/helpers/get-html-message.helper.js";
+import { type EmailOptions } from "~/libs/modules/email-service/libs/types/types.js";
 import { type Encryptor } from "~/libs/modules/encryptor/encryptor.js";
 import { HTTPCode } from "~/libs/modules/http/http.js";
 import { token } from "~/libs/modules/token/token.js";
 import {
+	type ForgotPasswordRequestDto,
+	type ResetPasswordRequestDto,
 	type UserSignInRequestDto,
 	type UserSignInResponseDto,
 	type UserSignUpRequestDto,
 	type UserSignUpResponseDto,
+	type VerifyTokenRequestDto,
 } from "~/modules/users/libs/types/types.js";
 import { type UserService } from "~/modules/users/user.service.js";
 
+import { getDefaultExpirationDate } from "../password-token/libs/helpers/get-default-expiration-date.helper.js";
+import { type PasswordTokenService } from "../password-token/password-token.service.js";
 import { UserValidationMessage } from "./libs/enums/enums.js";
 import { AuthorizationError } from "./libs/exceptions/exceptions.js";
 
 class AuthService {
+	private emailService: EmailService;
+
 	private encryptor: Encryptor;
+
+	private passwordTokenService: PasswordTokenService;
+
 	private userService: UserService;
 
-	public constructor(userService: UserService, encryptor: Encryptor) {
+	// eslint-disable-next-line max-params
```

Comment:
I think you can easily avoid the error if you pass parameters to the constructor as a "bag of properties"
I mean `{
		userService: UserService,
		encryptor: Encryptor,
		emailService: EmailService,
		passwordTokenService: PasswordTokenService,
}`